packages/product-feed-luigis-box/src/Model/FeedItem/LuigisBoxProductFeedItemFactory.php

```diff
@@ -125,6 +126,20 @@ protected function getPrice(Product $product, DomainConfig $domainConfig): Price
         );
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Product\Product $product
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domainConfig
+     * @return \Shopsys\FrameworkBundle\Model\Pricing\Price
+     */
+    protected function getBasicPrice(Product $product, DomainConfig $domainConfig): Price
+    {
+        return $this->productPriceCalculationForCustomerUser->calculateBasicPriceForCustomerUserAndDomainId(
+            $product,
+            $domainConfig->getId(),
+            null,
```

Comment:
it is not necessary to pass `null` here, it is a default value

Reply:
btw, the same applies to `getPrice()`  above where `calculatePriceForCustomerUserAndDomainId()` is called :slightly_smiling_face: